config/config_test.go

```diff
@@ -77,10 +77,11 @@ func TestNew(t *testing.T) {
 			want: &Config{
 				Version: "v2.0.0",
 				Server: Server{
-					Port:            8082,
-					Timeout:         "10s",
-					ShutdownTimeout: "10s",
-					ShutdownDelay:   "9s",
+					Port:              8082,
+					Timeout:           "10s",
+					ShutdownTimeout:   "10s",
+					ShutdownDelay:     "9s",
+					DisableKeepAlives: false,
```

Comment:
```suggestion
					DisableKeepAlives: true,
```
初期値のことを考えて、trueでテストした方が良いと思いました

Reply:
fix: [fa1ff08](https://github.com/AthenZ/authorization-proxy/pull/41/commits/fa1ff08cbdc4cdb7b1a465aa59680cf9198996ef)